backend/sources/google/events-router/src/main/java/co/airy/core/sources/google/EventInfo.java

```diff
@@ -19,4 +23,20 @@
     private Channel channel;
     private Long timestamp;
     private boolean isMessage;
+
+    @JsonIgnore
+    public Map<String, String> getMessageHeaders() {
+        final Map<String, String> headers = new HashMap<>();
+
+        final JsonNode suggestionResponse = event.getSuggestionResponse();
+        if (suggestionResponse != null) {
+            if (suggestionResponse.get("postbackData") != null) {
+                headers.put("postback.payload", suggestionResponse.get("postbackData").textValue());
```

Comment:
This header is picked up by the webhook publisher. So the postback payload will simply show up in the webhook payload